fre/pp/checkout_script.py

```diff
@@ -73,8 +73,10 @@ def checkout_template(experiment = None, platform = None, target = None, branch
         if current_tag == git_clone_branch_arg or current_branch == git_clone_branch_arg:
             fre_logger.info(f"checkout exists ('{directory}/{name}'), and matches '{git_clone_branch_arg}'")
         else:
-            fre_logger.info(f"ERROR: checkout exists ('{directory}/{name}') and does not match '{git_clone_branch_arg}'")
-            fre_logger.info(f"ERROR: current branch is '{current_branch}', current tag-describe is '{current_tag}'")
+            fre_logger.info(
```

Comment:
This error message should go to `logger.error` instead of `logger.info` but you only changed the spacing, and it's not related to cmor. INFO messages can be silenced with `--quiet` but ERROR messages should never be suppressed.